src/main/resources/modules/metabolic_syndrome_disease.json

```diff
@@ -1,799 +1,1033 @@
-                   "It is estimated that 27.8% of people with diabetes are undiagnosed",
-                   "https://www.cdc.gov/diabetes/pdfs/data/2014-report-estimates-of-diabetes-and-its-burden-in-the-united-states.pdf",
-                   "therefore the diabetes %s are all scaled up by a factor of 1.278, ",
-                   "so that the total % of people that eventually get diabetes is higher, but the current population with diabetes should be around 8.8%",
-                   "because there is a delay before diabetes onsets",
-                   "similarly the prediabetes #s are scaled up (but by a little less) so the current prediabetic % at any time is ~38%"],
-      "complex_transition" : [
-        {
-          "condition" : {
-            "condition_type" : "Race",
-            "race" : "White"
+      "remarks": [
+        "probability: 0.296 # (1.0==100%) http://www.cdc.gov/MMWr/preview/mmwrhtml/su6203a24.htm#Tab"
+      ]
+    },
+    "Onset_Hypertension": {
+      "type": "SetAttribute",
+      "attribute": "hypertension",
+      "value": true,
+      "direct_transition": "Diabetes_Prevalence"
+    },
+    "No_Hypertension": {
+      "type": "SetAttribute",
+      "attribute": "hypertension",
+      "value": false,
+      "direct_transition": "Diabetes_Prevalence"
+    },
+    "Diabetes_Prevalence": {
+      "type": "Simple",
+      "remarks": [
+        "diabetes prevalence %s based on the following",
+        "http://www.mass.gov/eohhs/gov/departments/dph/programs/community-health/diabetes/facts/diabetes-statistics.html",
+        "prediabetes prevalence is ~38 % overall and similar among most races but slightly lower among Native americans",
+        "https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3830901/",
+        "http://www.diabetes.org/living-with-diabetes/treatment-and-care/high-risk-populations/treatment-american-indians.html",
+        "It is estimated that 27.8% of people with diabetes are undiagnosed",
+        "https://www.cdc.gov/diabetes/pdfs/data/2014-report-estimates-of-diabetes-and-its-burden-in-the-united-states.pdf",
+        "therefore the diabetes %s are all scaled up by a factor of 1.278, ",
+        "so that the total % of people that eventually get diabetes is higher, but the current population with diabetes should be around 8.8%",
+        "because there is a delay before diabetes onsets",
+        "similarly the prediabetes #s are scaled up (but by a little less) so the current prediabetic % at any time is ~38%"
+      ],
+      "complex_transition": [
+        {
+          "condition": {
+            "condition_type": "Race",
+            "race": "White"
           },
-          "distributions" : [
-            { "distribution" : 0.083, "transition" : "Eventual_Diabetes", "remarks" : ["0.065 * 1.278"] },
-            { "distribution" : 0.45, "transition" : "Eventual_Prediabetes", "remarks" : ["0.38 * 1.2"] },
```

Comment:
A note just for future consideration: are there ways to keep this kind of thing on one line? I feel like the one-liner is a lot easier to follow than the exploded version here.

Reply:
There are ways of doing this, but it would be a little messy.  I can think of a creative way of using `toJSON` within objects using `JSON.stringify` to replace the distribution contents with a keyword, which we then replace with our custom format after it has been turned into a string.  Maybe something worth adding a low priority issue for.